curriculum/challenges/english/08-coding-interview-prep/data-structures/adjacency-matrix.english.md

```diff
@@ -9,9 +9,10 @@ forumTopicId: 301621
 <section id='description'>
 
 Another way to represent a graph is to put it in an <dfn>adjacency matrix</dfn>.
-An <dfn>adjacency matrix</dfn> is a two-dimensional (2D) array where each nested array has the same number of elements as the outer array. In other words, it is a matrix or grid of numbers, where the numbers represent the edges. Zeros mean there is no edge or relationship.
+An <dfn>adjacency matrix</dfn> is a two-dimensional (2D) array where each nested array has the same number of elements as the outer array. In other words, it is a matrix or grid of numbers, where the numbers represent the edges. 
+<strong>Note</strong>: The numbers to the top and left of the matrix are just labels for the nodes and the Ones means there exists an edge between the vertices (nodes) representing the row and column and Zeros mean there is no edge or relationship.
```

Comment:
```suggestion

<strong>Note</strong>: The numbers to the top and left of the matrix are just labels for the nodes. Inside the matrix ones mean there exists an edge between the vertices (nodes) representing the row and column.  Finally, zeros mean there is no edge or relationship.
```
One and zero are not a proper nouns - no need to capitalise them.  I also split the sentences to aide comprehension.